src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -1378,7 +1381,10 @@ equalityExpression
 
 // boolean relational expressions (level 5)
 relationalExpression
-    :    shiftExpression ( "instanceof"^ typeSpec[true])?
+    :   shiftExpression ( "instanceof"^  (
```

Comment:
Why did `shift` lose a space?

Reply:
Fixed.

Reply:
This isn't fixed. You can still see difference in view.
https://github.com/checkstyle/checkstyle/pull/8294/files#diff-cb42b9eb592a12d0fe72f51d1ccd501bL1381

Reply:
Shoot, wrong shiftExpression. Fixing now.